components/Pages/BrokerAssistant/SearchServiceUser/index.js

```diff
@@ -0,0 +1,127 @@
+import { useState } from 'react';
+import { BROKER_PORTAL_ROUTE, CARE_PACKAGE_ROUTE, LOGOUT_ROUTE } from '../../../../routes/RouteConstants';
+import { Button, Container, Header, Input, Breadcrumbs } from '../../../HackneyDS'
+import DatePicker from '../../../HackneyDS/DatePicker';
+import ServiceUserDetails from '../../BrokerPortal/ServiceUserDetails';
+import AlternativePagination from '../../../AlternativePagination';
+import Loading from '../../../Loading';
+import FormGroup from '../../../HackneyDS/FormGroup';
+import SearchResult from '../../../SearchResult';
+
+const links = [
+  { text: 'Broker Portal', href: BROKER_PORTAL_ROUTE },
+  { text: 'Care Charges', href: CARE_PACKAGE_ROUTE },
+  { text: 'Log Out', href: LOGOUT_ROUTE },
+];
+
+const breadcrumbs = [
+  { text: 'Home', href: '/' },
+  { text: 'Broker Portal', href: BROKER_PORTAL_ROUTE },
+  { text: 'Search for a service user' },
+];
+
+const SearchServiceUser = ({
+  searchResults,
+  pageNumber,
+  setPageNumber,
+  totalCount,
+  totalPages,
+}) => {
+  const [initialFilters] = useState({
+    postcode: '',
+    firstName: '',
+    lastName: '',
+    hackneyId: '',
+    dateOfBirth: null,
+  });
+  const [filters, setFilters] = useState({ ...initialFilters });
+  const [searchFilters, setSearchFilters] = useState({ ...initialFilters });
+
+  const changeFilters = (field, value) => {
+    setFilters(prevState => ({
+      ...prevState,
+      [field]: value
+    }));
+  };
+
+  const allocateToBroker = (item) => {
+    alert(item);
+  };
+  const viewPackageHistory = (item) => {
+    alert(item);
+  };
+
+  const onSearch = () => setSearchFilters({ ...filters });
+
+  return (
+    <Container className="search-service-user">
+      <Header links={links}/>
+      <Container maxWidth='1080px' margin='0 auto' padding="8px 60px 0 60px">
+        <Breadcrumbs values={breadcrumbs}/>
+        <Container padding="60px 0 0">
+          <h3 className="search-service-user__title">Search for a service user</h3>
+          <Container className="search-service-user__filters">
+            <Input
```

Comment:
4 inputs below are basically the same. You can create an array of input like `const inputs = [{ key: 'firstName', label: 'First Name' }, ...]` outside of component and map it here. It will be much cleaner 
```suggestion
            inputs.map(input => (
            <Input
              key={input.label}
              value={filters[input.key]}
              onChangeValue={value => changeFilters(key, value)}
              label={input.label}
            />
            ))
```